lib/pbench/server/api/resources/users_api.py

```diff
@@ -126,6 +127,7 @@ def post(self):
                 first_name=first_name,
                 last_name=last_name,
                 email=email_id,
+                registered_on=datetime.datetime.now(),
```

Comment:
Yeah, the advantage of the explicit `__init__` is encapsulating and isolating automatic stuff like this; I really don't like the idea of relying on getting this right anyplace we create a user. (Although there probably won't be many, the knowledge really belongs inside the `User` model, not here.)

Note that it's not actually necessary to _use_ any `kwargs` in the `__init__` ... just to pass them on to the super-constructor. But a better way might be to use [default](https://docs.sqlalchemy.org/en/14/core/defaults.html#python-executed-functions) in the Column declaration to call a method (or maybe even just `default=datetime.datetime.now()` would work).

Reply:
yes adding a `default=` is better